packages/doi2bib/doi2bib.0.4.0/opam

```diff
@@ -14,7 +14,9 @@ depends: [
   "cmdliner" {>= "1.0.0"}
   "decompress" {>= "1.3.0"}
   "ezxmlm" {>= "1.1.0"}
-  "tls" {>= "0.12.0"}
+  "lwt" {>= "5.3.0"}
+  "bigstringaf" {>= "0.2.0"}
```

Comment:
```suggestion
  "bigstringaf" {>= "0.2.0"}
  "uri" {>= "4.0"}
```
bigstringaf is implicitly linked from uri so the dependency itself does not work for the lower bound